md/release-notes.md

```diff
@@ -364,6 +364,22 @@ Please use the Tomcat bundle instead, or see the [Custom Deployment](deploy-bund
 ## Bug fixes
 ### Fixes in Documentation
 * BS-19323 Bonita documentation is not clear enough about PostgreSQL versions supported
+* BST-151 Default outgoing transition not executed when  there is more than one outgoing transition from inclusive gateway
+* BST-269 Document workaround for macos - ssh issue
+* BST-171 Can't share a project on github using HTTPS and double authentication factor
+* BST-341 On Mac, Studio does not start if recent Java versions installed
```

Comment:
Would you mind to change the description of the BST-341 ? As far as we know the encountered issue is more related to JRE configurations than to `recent Java versions installed`, I think it would be a terrible idea to write this in the release note ...

Reply:
Sure.

Reply:
@alachambre The procedure is more "you change the Issue title and then you notify that it has changed", so that the release notes and the Jira title are consistent